defaults/main.yml

```diff
@@ -1,8 +1,25 @@
 # SPDX-License-Identifier: MIT
 ---
-# Here is the right place to put the role's input variables.
-# This file also serves as a documentation for such a variables.
+# List of containers and systemd units to define
+podman_containers: []
 
-# Examples of role input variables:
-podman_foo: foo
-podman_bar: bar
+# If this is true, the role will automatically create the host directories
+# specified in volume or mount.
+podman_create_host_directories: false
```

Comment:
More a general question.  Is it needed/conventional to use the `podman_` prefix?  It looks redundant to me but I lack Ansible experience.

Reply:
It is according to ansible best practises to prefix variables by role name.

Reply:
Yes - all global variables should have the `ROLENAME_` prefix - see https://github.com/redhat-cop/automation-good-practices/tree/main/roles#naming-things